src/core/01_network_pair.tf

```diff
@@ -0,0 +1,36 @@
+resource "azurerm_resource_group" "rg_pair_vnet" {
+  name     = format("${local.project_pair}-vnet-rg", )
```

Comment:
Please remove the format, can be an error

Reply:
done